deployment/admission-webhook-datree.yaml

```diff
@@ -58,7 +58,7 @@ spec:
             requests:
               cpu: 100m
               memory: 256Mi
-          image: datree/admission-webhook:0.1.8
+          image: datree/webhook-staging:a7437b8
```

Comment:
this should not be deployed to production, or else users will run on staging

Reply:
it wouldn't, the release_to_prod workflow will update it with the values from the prod YAML

Reply:
Why it has to be staging? ATM the install script takes the file from main branch. Maybe we need to change the install script also to get the yaml from s3 and not main - if you already added this cool thing that updates the file